pkg/cmd/admin/admin.go

```diff
@@ -27,15 +27,15 @@ import (
 	exipfailover "github.com/openshift/origin/pkg/cmd/experimental/ipfailover"
 	"github.com/openshift/origin/pkg/cmd/server/admin"
 	"github.com/openshift/origin/pkg/cmd/templates"
+	. "github.com/openshift/origin/pkg/cmd/templates"
```

Comment:
Don't use `.`, just use `templates.LongDesc`.

I'd also prefer that we be using heredoc is combination with this, where you do auto unwrapping of paragraphs.